### PR TITLE
SIGNL4 node added.

### PR DESCRIPTION
**Pull request by rons4**
I hope I did everything correctly.

### CODE REVIEW

File: packages/nodes-base/nodes/SIGNL4/GenericFunctions.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+			method: 'POST',
+			body: message,
+			qs: {},
+			uri: `https://connect.signl4.com/webhook/${teamsecret}`,
+			json: true
+		});
+	}
+	catch (error) {
+		
+		throw new Error(`I am behind. Status: ${JSON.stringify(teamsecret)}`);
```
Comment (janober) [inline packages/nodes-base/nodes/SIGNL4/GenericFunctions.ts:39]: Do not understand this part. What does it mean "I am behind...?" and why does it output the "secret" access token?

Why is there any more code afterward that would never get called?

File: packages/nodes-base/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "n8n-nodes-base",
-  "version": "0.65.0",
+  "version": "0.66.0",
```
Comment (janober) [inline packages/nodes-base/package.json:3]: Version numbers do only get changed by us when we release a new version.

Comment (rons4) [reply packages/nodes-base/package.json:3]: OK, got it.

**Comment by janober** [comment]
Great, thanks a lot. Just got merged.

**Comment by janober** [comment]
Got released with n8n@0.71.0

**Pull request merged**